9.1/docker-entrypoint.sh

```diff
@@ -15,7 +15,7 @@ if [ "$1" = 'postgres' ]; then
 
 	# look specifically for PG_VERSION, as it is expected in the DB dir
 	if [ ! -s "$PGDATA/PG_VERSION" ]; then
-		gosu postgres initdb
+		gosu postgres initdb "$POSTGRES_INITDB_ARGS"
```

Comment:
I think this will cause trouble if we need multiple arguments (ie, something like `POSTGRES_INITDB_ARGS='--a --b --c --d="e f"`).

Something like this is probably what we'll have to do instead: (so we get quote parsing, etc as appropriate)
```bash
eval "gosu postgres initdb $POSTGRES_INITDB_ARGS"
```